src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlEventSourceListener.netfx.cs

```diff
@@ -181,18 +181,14 @@ private void OnEndExecute(EventWrittenEventArgs eventData)
                 if (activity.IsAllDataRequested)
                 {
                     int compositeState = (int)eventData.Payload[1];
-                    if ((compositeState & 0b001) == 0b001)
-                    {
-                        activity.SetStatus(Status.Unset);
-                    }
-                    else if ((compositeState & 0b010) == 0b010)
+                    if ((compositeState & 0b010) == 0b010)
```

Comment:
This is changing the logic

Reply:
Right, in the case when compositeState is 0b011, this if statement is required even if Unset is the default. Thanks!